README.md

```diff
@@ -50,28 +50,40 @@ This project is a follow-up to the now unmaintained (and closed-source) [Message
 - Open source, [AGPL License](/LICENSE).
 
 ## User Guide
-
-- Tap, or slide / swipe to type a letter.
-- Swipe up or down on `A` to capitalize. (If you changed the layout, the button next to `#`) <img width=60px height=60px src="https://i.postimg.cc/Znt2Ft9G/thumbkey-1-1.png">
-- Double tap the space bar to type a comma, triple tap to type a period. More taps have more punctuation.
-- Press the `#` key to see a numeric keypad, and additional symbols.
-- Press the Settings Gear key to configure Thumb-Key.
-- Slide left on the backspace key to delete whole words.
-- Slide left or right on the spacebar to move the cursor.
-- Slide left on the `#` key to switch to a default language.
-- Slide up on the `#` key to select and copy all.
-- Slide down on the `#` key to paste.
-- Slide right on the `#` key to switch keyboard position.
-- Practice typing, and check your words per minute, using [monkeytype.com](https://monkeytype.com)
+- **Tap**, or **swipe** to type a letter.
+- **Swipe up** or **down** on `A` to capitalize. (If you changed the layout, the button next to `#`)  
+  <img width=60px height=60px src="https://i.postimg.cc/Znt2Ft9G/thumbkey-1-1.png">  
+- **Double tap** the space bar to type a comma, triple tap to type a period. More taps have more punctuation.
+- **Swipe left** on the backspace key to delete whole words.
+- **Swipe left** or **right** on the spacebar to move the cursor by 1 character.
+
+### Emoji Key
```

Comment:
You need a newline here, after every section header. Better yet run `prettier -w README.md` after you've made some changes. 

I'll add a lint for this in the CI to double check.